crates/swc/tests/fixture/issues-4xxx/4897/output/index.ts

```diff
@@ -1,19 +1,26 @@
 define([
     "require",
     "exports"
-], function(require, _exports) {
+], function(require, exports) {
     "use strict";
-    Object.defineProperty(_exports, "__esModule", {
+    Object.defineProperty(exports, "__esModule", {
         value: true
     });
-    _exports.boo = boo;
-    _exports.Foo = void 0;
+    function _export(target, all) {
+        for(var name in all)Object.defineProperty(target, name, {
+            get: all[name],
+            enumerable: true
+        });
+    }
+    _export(exports, {
```

Comment:
We need a helper for this

Reply:
The `_export` is always inlined if needed.
Do you prefer a helper for it?

Reply:
No I missed the declaration